### PR TITLE
Refresh pinnings

### PR DESCRIPTION
**Pull request by jakirkham**
Some of the pinnings here were out-of-date. This just refreshes them.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,29 +13,29 @@ source:
...
   run:
-    - boost-cpp 1.64.*
+    - boost-cpp 1.66.0
     - libgsasl 1.8.0 1
     - libntlm
     - libuuid
     - libxml2 2.9.*
-    - krb5
+    - krb5 1.14.*
     - curl
```
Comment (jakirkham) [inline recipe/meta.yaml:38]: Noticed `curl` was in `run`, but not `build`. Should it be in `build`?

**Comment by martindurant** [comment]
@jakirkham , can you check if #16 also works?

**Comment by jakirkham** [comment]
Not ATM. Am in the midst of updating several feedstocks in preparation for cb3.

**Comment by jakirkham** [comment]
That said, feel free to merge whichever first.

**Comment by martindurant** [comment]
Sorry, I'll require a merger, unfortunately.

**Comment by jakirkham** [comment]
Happy to. Please let me know if that looks ok.

**Pull request merged**